pages/products_page.py

```diff
@@ -23,4 +23,5 @@ def page_heading(self):
     def click_product(self, product):
         self.selenium.find_element(By.LINK_TEXT, product).click()
         from pages.home_page import CrashStatsHomePage
-        return CrashStatsHomePage(self.testsetup, product)
+        # Not sure what this does? 'product' ?
+        return CrashStatsHomePage(self.base_url, self.selenium, product)
```

Comment:
The `CrashStatsHomePage` takes a `product` argument, and if it's not `None` it opens the product's page directly.